libs/category/src/drivers/magento/transformers/category-page-config-transformer.service.ts

```diff
@@ -27,7 +27,7 @@ export class DaffMagentoCategoryPageConfigTransformerService {
   transformFilter(filter: FilterNode): DaffCategoryFilter {
     return {
       name: filter.name,
-      type: filter.__typename,
+      type: DaffCategoryFilterTypes.Equal,
```

Comment:
This will be hardcoded until a future PR where we make a separate aggregate type request, because in magento this is a separate request.